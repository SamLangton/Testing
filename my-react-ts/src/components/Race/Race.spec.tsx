/*
	Snapshot tests for "./Race.tsx"
*/
import * as React from 'react';
import Race from './Race';
import * as renderer from 'react-test-renderer';
jest.mock('../Race', () => 'Race');
describe('./Race.tsx', () => {

	it('should render as expected', () => {
		
		beforeEach(() => {
			require('Race').src = "https://avatars1.githubusercontent.com/u/16746699?v=4";
		});
		const component = renderer.create(
			<Race raceInProgress = {false}
			raceStartPosition={true}
			setWinner = {''} 
			/>
		);
		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});