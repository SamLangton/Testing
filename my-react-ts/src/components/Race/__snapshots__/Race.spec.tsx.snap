// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`./Race.tsx should render as expected 1`] = `
<div
  className="Race"
>
  <div>
    <div
      className="Track"
    >
      <div
        className="Horse"
        style={
          Object {
            "paddingLeft": "0px",
          }
        }
      >
        <img
          height="100px"
          src="https://avatars0.githubusercontent.com/u/5137591?v=4"
        />
      </div>
    </div>
  </div>
  <div>
    <div
      className="Track"
    >
      <div
        className="Horse"
        style={
          Object {
            "paddingLeft": "0px",
          }
        }
      >
        <img
          height="100px"
          src="https://avatars0.githubusercontent.com/u/6842205?v=4"
        />
      </div>
    </div>
  </div>
  <div>
    <div
      className="Track"
    >
      <div
        className="Horse"
        style={
          Object {
            "paddingLeft": "0px",
          }
        }
      >
        <img
          height="100px"
          src="https://avatars2.githubusercontent.com/u/2323810?v=4"
        />
      </div>
    </div>
  </div>
  <div>
    <div
      className="Track"
    >
      <div
        className="Horse"
        style={
          Object {
            "paddingLeft": "0px",
          }
        }
      >
        <img
          height="100px"
          src="https://avatars2.githubusercontent.com/u/1482046?v=4"
        />
      </div>
    </div>
  </div>
  <div>
    <div
      className="Track"
    >
      <div
        className="Horse"
        style={
          Object {
            "paddingLeft": "0px",
          }
        }
      >
        <img
          height="100px"
          src="https://avatars3.githubusercontent.com/u/15605543?v=4"
        />
      </div>
    </div>
  </div>
</div>
`;
